Makefile

```diff
@@ -11,8 +11,9 @@ setup:
 	sleep 10
 	docker exec ziggurat_kafka /opt/bitnami/kafka/bin/kafka-topics.sh --create --topic $(topic) --partitions 3 --replication-factor 1 --zookeeper ziggurat_zookeeper
 	docker exec ziggurat_kafka /opt/bitnami/kafka/bin/kafka-topics.sh --create --topic $(another_test_topic) --partitions 3 --replication-factor 1 --zookeeper ziggurat_zookeeper
+
 test: setup
```

Comment:
we should just have one step for CI - `test-cluster`

Why?
Because CI should prove that Ziggurat works for a kafka cluster. Kafka (except dev environments) never runs on a single broker.

While, we can keep `test` for development purpose. Additionally, we can create a task in Makefile, something like `test-local` which uses TESTING_TYPE=local

Reply:
@mjayprateek we already have job in gh actions for testing a cluster `run_tests_with_kafka_cluster`. for `local` are we only testing for a single broker?

Reply:
done